distribution/changelog.txt

```diff
@@ -31,6 +31,7 @@
 - Fix: [#19767] No message when path is not connected to ride exit and is therefore unreachable for mechanics.
 - Fix: [#19800] Crash when displaying station stats with more than 62 stations.
 - Fix: [#19801] The in-game load/save window cannot be resized anymore.
+- Fix: [#19832] Scenery Search
```

Comment:
Looks a bit too short

Reply:
I titled it after the open issue that this addresses. Would you prefer something more descriptive?

Reply:
I've updated the changelog entry.